mobile/lib/providers/backup/backup.provider.dart

```diff
@@ -297,23 +312,24 @@ class BackupNotifier extends StateNotifier<BackUpState> {
     final Set<BackupCandidate> assetsFromExcludedAlbums = {};
 
     for (final album in state.selectedBackupAlbums) {
-      final assetCount = await album.albumEntity.assetCountAsync;
+      final assetCount = await ref
+          .read(mediaRepositoryProvider)
+          .countAssetsInAlbum(album.album.localId!);
```

Comment:
why do all the methods here need ref.read(repositoryProvider)? Can they eventually just use the interface, passed into the constructor?